site/src/sphinx/server-annotated-service.rst

```diff
@@ -801,7 +801,7 @@ more response types which can be used in the annotated service.
 - :api:`HttpResult`
 
   - It contains the :api:`HttpHeaders` and the object which can be converted into HTTP response body by
-    response converters. A user can customize the HTTP status and headers including the trailing headers,
+    response converters. A user can customize the HTTP status and headers including the trailers,
     with this type.
```

Comment:
not relevant to this PR but could you fix the indentation?

Reply:
Moved to the above line. :-)